plugins/chat/lib/extensions/user_notifications_extension.rb

```diff
@@ -28,7 +28,7 @@ def chat_summary(user, opts)
     return if @messages.empty?
     @grouped_messages = @messages.group_by { |message| message.chat_channel }
     @grouped_messages =
-      @grouped_messages.select { |channel, _| guardian.can_see_chat_channel?(channel) }
+      @grouped_messages.select { |channel, _| guardian.can_join_chat_channel?(channel) }
```

Comment:
same question about tests

Reply:
Because guardian is tested, I added a test which ensures that guardian is invoked